server/pools/views.py

```diff
@@ -30,3 +26,27 @@ def login(request):
 
 def reservation(request):
     pass
+
+
+def pools_list(request):
+    if request.method == 'GET':
+        pools = list(Pool.objects.values())
+        for p in pools:
+            del p['id']
```

Comment:
Not sure if removing database pool ids from response is necessary. In the further development it might be actually helpful.